src/stores/RoomViewStore.tsx

```diff
@@ -183,6 +184,12 @@ export class RoomViewStore extends Store<ActionPayload> {
                     wasContextSwitch: false,
                 });
                 break;
+            case Action.ViewFavouriteMessages:
```

Comment:
This can be added to the two existing cases like this:

```
case 'view_welcome_page':
case Action.ViewHomePage:
case Action.ViewFavouriteMessages:
    this.setState({
        roomId: null,
        roomAlias: null,
        viaServers: [],
        wasContextSwitch: false,
    });
    break;
```